README.md

```diff
@@ -12,12 +12,12 @@ using [Java Embedded Python](https://github.com/ninia/jep).
 ### Generic Python
 [![Maven metadata URI](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/feedzai/openml-generic-python/maven-metadata.xml.svg)](https://mvnrepository.com/artifact/com.feedzai/openml-generic-python)
 
-You can find in the `openml-generic-python` module a provider that allows
-users to load in code that conforms to a simple API.
+The `openml-generic-python` module contains a provider that allows
+developers to load code that conforms to a simple API.
```

Comment:
maybe update to `developers to load Python code` ? It's implicit that the code loaded should be in the Python programming language, but better explicit than implicit, do you agree?